src/scales/quantitativeScale.ts

```diff
@@ -42,6 +51,44 @@ module Plottable {
       return super.domain(values); // need to override type sig to enable method chaining:/
     }
 
+    /**
+     * Gets the minimum domain value when autoDomain()-ing. 
+     */
+    public min(): D;
+    /**
+     * Sets the minimum domain value when autoDomain()-ing.
+     * 
+     * @returns {QuantitativeScale<D>} The calling QuantitativeScale.
+     */
+    public min(value: D): QuantitativeScale<D>;
+    public min(value?: D): any {
+      if (value === undefined) {
```

Comment:
`.parent()` on `Component` takes `null` as a valid param (but I also wrote that).

We do have a lot of places in our codebase with `=== undefined`, but I think that's an error. I'll make a ticket.